.github/workflows/test.yml

```
@@ -90,12 +103,12 @@ jobs:
         if: github.event_name != 'schedule'
         run: shellcheck -o all -S style -s sh $(find . -iname "*.sh")
 
-  # This job reports the results of the test matrix above
+  # This job reports the results of the matrixes above
   test:
     if: always()
-    needs: test-matrix
+    needs: [test-matrix, all-features, oldstable]
```

```suggestion
    needs: [rustfmt, test-matrix, all-features, oldstable, deno-checks, shellcheck]
```
Shouldn't all test jobs in this workflow be included here...to keep from having to add them individually to the required status checks list in the `main` branch protection rule?